src/pages/ServiceDetails/ServiceInfo.tsx

```diff
@@ -181,34 +186,50 @@ class ServiceInfo extends React.Component<ServiceDetails, ServiceInfoState> {
                     </NavItem>
                   </Nav>
                   <TabContent>
-                    <TabPane eventKey={'workloads'}>
+                    <WithErrorBoundary
+                      component={TabPane}
+                      eventKey={'workloads'}
+                      message={this.errorBoundaryMessage('Workloads')}
+                    >
                       {(Object.keys(workloads).length > 0 || this.props.serviceDetails.istioSidecar) && (
                         <ServiceInfoWorkload workloads={workloads} namespace={this.props.namespace} />
                       )}
-                    </TabPane>
-                    <TabPane eventKey={'sources'}>
+                    </WithErrorBoundary>
+                    <WithErrorBoundary
+                      component={TabPane}
+                      eventKey={'sources'}
+                      message={this.errorBoundaryMessage('Sources')}
+                    >
                       {(Object.keys(dependencies).length > 0 || this.props.serviceDetails.istioSidecar) && (
                         <ServiceInfoRoutes dependencies={dependencies} />
                       )}
-                    </TabPane>
-                    <TabPane eventKey={'virtualservices'}>
+                    </WithErrorBoundary>
+                    <WithErrorBoundary
+                      component={TabPane}
+                      eventKey={'virtualservices'}
+                      message={this.errorBoundaryMessage('Virtual Services')}
```

Comment:
"Virtual Services" -> "VirtualServices"
I think it should be one single word as it's a term.